src/main/java/com/bc/demoproj/DemoprojApplication.java

```diff
@@ -1,33 +1,33 @@
-package com.bc.demoproj;
-
-import java.util.Arrays;
-
-import org.springframework.boot.CommandLineRunner;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.annotation.Bean;
-
-@SpringBootApplication
-public class DemoprojApplication {
-
-  public static void main(String[] args) {
-    SpringApplication.run(DemoprojApplication.class, args);
-  }
-
-  @Bean
-  public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
-    return args -> {
-
-      System.out.println("Let's inspect the beans provided by Spring Boot:");
-
-      String[] beanNames = ctx.getBeanDefinitionNames();
-      Arrays.sort(beanNames);
-      for (String beanName : beanNames) {
-        System.out.println(beanName);
-      }
-
-    };
-  }
-
-}
+//package com.bc.demoproj;
```

Comment:
We should avoid disabled source code by comments. Let's remove such a piece of code, instead.